.github/workflows/publish.yaml

```diff
@@ -54,6 +59,11 @@ jobs:
           # Make the version available as env variable for next steps
           echo CURRENT_VERSION=$CURRENT_VERSION >> $GITHUB_ENV
         shell: bash
+      - name: Build dist
+        run: |
+          python setup.py sdist bdist_wheel
+      - name: Publish to PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1
```

Comment:
Small suggestion: you could enable debug verbosity and printing of hashes, just in case.

https://github.com/pypa/gh-action-pypi-publish?tab=readme-ov-file#for-debugging

Reply:
Good idea